account-compression/Anchor.toml

```
@@ -0,0 +1,26 @@
+[features]
+seeds = false
+
+[test.validator]
+url = "http://127.0.0.1:8899"
+
+[programs.localnet]
+psp_account_compression = "DmtCHY9V1vqkYfQ5xYESzvGoMGhePHLja9GQ994GKTTc"
+
+[programs.testnet]
+psp_account_compression = "DmtCHY9V1vqkYfQ5xYESzvGoMGhePHLja9GQ994GKTTc"
```

Interesting that you put the account compression program into it's own anchor project. I would have put it into the other program folder, but I think it makes sense since this is a public good and we might want to put it into it's own repo eventually.

About naming, let's ditch the psp prefix since this will be not a light protocol utility but just the new compression program.

```suggestion
account_compression = "DmtCHY9V1vqkYfQ5xYESzvGoMGhePHLja9GQ994GKTTc"
```

---

I can't put it in the same programs/ project, because that would cut off the possibility of using different Anchor CLIs.

If you move this program together with all the others, it's going to be picked up by `$TOP_DIR/programs/Anchor.toml`, therefore `light-anchor build` and `npx nx run @lightprotocol/program:build` are going to pick it too. And then IDL for all programs would be generated by one specific Anchor version.

Only havng separate Anchor.toml files guarantees what you wanted - possibility of using different Anchor versions, CLIs, IDLs.